scientific_library/tvb/dsl/LEMS2python.py

```diff
@@ -106,4 +112,9 @@ def regTVB_templating(model_filename):
                 f.seek(0)
                 f.writelines(lines)
     except:
-        print('unable to add new model to __init__.py')
\ No newline at end of file
+        print('unable to add new model to __init__.py')
```

Comment:
Maybe you could use logging instead of print calls in the entire module ?

Reply:
In almost any other TVB module you should see calls to a logger which is defined at the top of the module; please replicate that pattern here.  

Also, no bare except, should be
```python
except IOError as e:
    logger.error('ioerror: %s', e)
```
since the error could be filesystem out of space, user presses ctrl-c, etc.

Reply:
has been updated as requested